### PR TITLE
Visualizer updates

### PR DESCRIPTION
**Pull request by y-ng**
- add switching b/w modalities on approve segmentation page
- add simple preview page for 2d renders
- some changes to what controls are displayed in the panel

### CODE REVIEW

File: src/app/segmentation/page.tsx [hunk 1]
```diff
@@ -1,23 +1,86 @@
...
+    } else if (modality == CONSTANTS.T2W) {
+      setModality(CONSTANTS.T1C);
+    }
+  };
+
   return <>
     <NavBar view="segmentation" />
     <h1>Approve Segmentation Images</h1>
+    <h2>Modality: {modality}</h2>
     <div className='flex justify-center mb-7' id='seg-preview'>
```
Comment (y-ng) [inline src/app/segmentation/page.tsx:69]: Note: I wanted to use one visualization component with modality param, but render wouldn't update on modality change; needed to use separate visualizations for each modality for it to work.

File: src/app/globals.css [hunk 2]
```diff
@@ -100,43 +109,47 @@ form label {
 }
 
 #my-gui-container {
-  width: 200px;
-  position: fixed;
-  top: 100px;
-  right: 10px;
+  width: 13rem;
+  position: absolute;
```
Comment (zoeyz101) [inline src/app/globals.css:113]: hmm if we are making the gui positioning absolute (which i think is fair) then we should make it so only the left info container is scrollable

File: src/app/globals.css [hunk 3]
```diff
@@ -100,43 +109,47 @@ form label {
...
-  padding-bottom: 20px;
+  padding-left: 0.5rem;
+  padding-top: 0.5rem;
 }
 
 .btn {
   cursor: pointer;
 }
+
+.dg ul {
```
Comment (zoeyz101) [inline src/app/globals.css:149]: be careful what we put in global vs in a component css cuz the import chain will be weird thats probably what caused the issues in the first place

File: src/app/search-bar/search-bar.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ function SearchBar() {
         </div>
         <input type="search" id="default-search" 
           className="block w-full mb-0 p-2 ps-10 text-sm text-gray-900 border border-gray-300 rounded-lg bg-gray-50 focus:ring-blue-500 focus:border-blue-500 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500" 
-          placeholder="Find other Patients" 
+          placeholder="Find other patients"
```
Comment (zoeyz101) [inline src/app/search-bar/search-bar.js:16]: fair lol

File: src/app/segmentation/page.tsx (hunk 1 above)
Comment (zoeyz101) [reply src/app/segmentation/page.tsx:69]: we might wanna add some loading later on so the page glitching thing isnt as bad (maybe some opaque box on top while loading happens in the back idk

File: src/app/globals.css (hunk 3 above)
Comment (y-ng) [reply src/app/globals.css:149]: yeah li components were grabbing styling from elsewhere
dg class seems to only be used in the control panel, so thought it'd be fine in global

**Review by zoeyz101 — APPROVED** [review]

File: src/app/segmentation/page.tsx (hunk 1 above)
Comment (y-ng) [reply src/app/segmentation/page.tsx:69]: good point, a later task it is

File: src/app/globals.css (hunk 2 above)
Comment (y-ng) [reply src/app/globals.css:113]: valid point, do you want to add that w the styling or maybe we can do it later

**Pull request merged**